version_control/Codurance_September2020/modules/Clients-Logos.module/module.html

```diff
@@ -2,6 +2,16 @@
 
 {{ require_css(get_asset_url("../../css/modules/clients-logos.css")) }}
 
+
+  <style>
+      .client-logos__logos-group{
+        {% if module.style.logo_color == "grey" %}
+        	filter: contrast(0);
+        {% endif %}    
+      }
+  </style>
```

Comment:
I think we should wrap this style block with the scope tag in order for it to apply only on the corresponding module instance (I know we normally only use it once on each page, but just in case!) 
```suggestion
{% require_css }
  <style>
    {% scope_css }
      .client-logos__logos-group{
        {% if module.style.logo_color == "grey" %}
        	filter: contrast(0);
        {% endif %}    
      }
    {% end_scope_css }
  </style>
{% require_css }
```

Reply:
I've tried this and it doesn't work for me... Have you checked on your end?